package.json

```diff
@@ -42,17 +42,19 @@
     "react-hook-form": "^7.34.2",
     "react-infinite-scroll-component": "^6.1.0",
     "react-redux": "^8.0.2",
+    "react-window": "^1.8.8",
     "sass": "^1.53.0"
   },
   "devDependencies": {
-    "@samandar.boymurodov/editorjs-codeflask": "1.0.12",
     "@editorjs/delimiter": "^1.2.0",
     "@editorjs/inline-code": "^1.3.1",
     "@editorjs/quote": "^2.4.0",
     "@next/bundle-analyzer": "^12.3.0",
+    "@samandar.boymurodov/editorjs-codeflask": "1.0.12",
```

Comment:
do we need this change?

Reply:
It is npm that may have sorted ordering after installing react-window.